src/sssom/io.py

```diff
@@ -159,20 +160,20 @@ def get_metadata_and_prefix_map(
     else:
         prefix_map = {}
     converter = Converter.from_prefix_map(prefix_map)
-    converter = _merge_converter(converter, prefix_map_mode=prefix_map_mode)
+    converter = _merge_converter(converter, mode=merge_mode)
```

Comment:
Why is `merge_mode` called `mode` in _merge_converter? Because the context is clear?

Reply:
Oversight. I was planning on deleting this method next Pr anyway since it’s only used once inside another private method